lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -235,10 +235,16 @@ static void checkInheritanceClause(
       // AnyObject is not allowed except on protocols.
       if (layout.hasExplicitAnyObject &&
           !isa<ProtocolDecl>(decl)) {
-        decl->diagnose(canHaveSuperclass
+            if(!layout.isAnyObject()){
+              decl->diagnose(diag::composition_of_protocol_and_AnyObject, typeDecl->getName() ,inheritedTy);            
+            }
+            else{
```

Comment:
Did you run `clang-format`? I think the usual llvm style is:
```
if () {
} else {
}
```

Reply:
Ran clang-format and conformed to the LLVM Style.